data-generation/database-query.js

```diff
@@ -0,0 +1,64 @@
+const Promise = require('bluebird');
+const fs = Promise.promisifyAll(require('fs'));
+const files = require('./files-index.js');
+const db = require('../database/queries.js');
+const { addDocumentsInBulk } = require('../elasticsearch/queries');
+
+const insertQueries = {
+  logs: 'INSERT INTO logs (user_id, date, "createdAt", "eventTypeId", "sessionId") VALUES',
+  views: 'INSERT INTO playlist_views (playlist_id, genre_id, date, "createdAt", "logId") VALUES',
+  searches: 'INSERT INTO searches (value, date, "createdAt", "logId") VALUES',
+  reactions: 'INSERT INTO song_reactions (song_id, liked, playlist_id, genre_id, date, "createdAt", "logId") VALUES',
+  responses: 'INSERT INTO song_responses (song_id, "listenedTo", playlist_id, genre_id, "date", "createdAt", "logId") VALUES'
+};
+
+const formatData = data => data.toString().replace(/,\n$/, '\n');
+
+const updateTable = (query, file) => (
+  fs.readFileAsync(file)
+    .then((data) => {
+      const parsed = formatData(data);
+      return db.queryDB(`${query} ${parsed}`);
+    })
+    .then(() => fs.truncateAsync(file))
+    .catch(err => console.log(`error updating table with file ${file}`, err))
+);
+
+const updateDatabase = () => (
```

Comment:
You could probably refactor this to execute updating the table in one query instead of multiple. Sequelize can take in an object with various properties, and if the keys all match the table fields, it will update all the values accordingly.